docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -161,36 +150,40 @@ kubectl get subscriptions lastorder-sub -o=jsonpath="{.status.ready}"
 
 The operation was successful if the returned status says `true`.
 
-  </details>
-  <details>
-  <summary label="Kyma Dashboard">
-  Kyma Dashboard
-  </summary>
-
-1. Go back to your `lastorder` Function's view in Kyma Dashboard.
-2. In the **Configuration** tab, click on **Create Subscription+**.
-3. Provide the following parameters:
-    - **Subscription name**: `lastorder-sub`
-    - **Application name**: `myapp`
-    - **Event name**: `order.received`
-    - **Event version**: `v1`
-
-    - **Event type** is generated automatically. For this example, it's `sap.kyma.custom.myapp.order.received.v1`. 
-    
-4. Click **Create**. 
-
   </details>
 </div>
 
 ## Trigger the workload with an event
 
-We created the `lastorder` Function, exposed it, and created a Subscription for it to listen for `order.received.v1` events. Now it's time to send such an event and trigger the Function. For the sake of this example, we'll port-forward the Kyma Eventing Service to localhost. 
+We created the `lastorder` Function, and created a Subscription for it to listen for `order.received.v1` events. Now it's time to send an event and trigger the Function. For the sake of this example, we'll port-forward the Kyma Eventing Service to localhost. 
```

Comment:
nit: Is `Eventing Service` the official name which is used in the documentation ? 
Otherwise I would propose to just use the name Event-Publisher-Proxy as this is also the name of the service :) (keep things simple)

Reply:
It was used before as `Kyma Eventing Service`, its the getting started guide, and at this stage we haven't explained about our architecture (what is eventing-controller and publisher proxy), so I say `Kyma Eventing Service` is fine. What do you think @kyma-project/technical-writers ?